src/Components/MainHeader/MainHeader.css

```diff
@@ -0,0 +1,29 @@
+.MainHeader {
```

Comment:
naming in css mustn't contain an uppercase character

for example `main-header`